src/cli/args.rs

```diff
@@ -88,6 +88,10 @@ pub enum Subcommand {
         /// Place results in a directory other than the current one
         #[arg(short = 'd', long = "dir", value_hint = ValueHint::FilePath)]
         output_dir: Option<PathBuf>,
+
+        /// Remove the source file after successful decompression
+        #[arg(short = 'r', long, default_value_t = false)]
```

Comment:
false is the default value for booleans right? :thinking:

Reply:
just want it to be more explicit. because remove file is a danger operation.

yes, I'm OK to remove that.